### ISSUE TITLE
About localization

### ISSUE THREAD

**Issue by ExtremoDuro** [MassiveCraft/Factions#1032]
What's the best way to translate the whole plugin to other languages?
I am interested in translating it completely to spanish. Are there specific files for the strings? Should I fork the plugin to translate it? Has anyone tried to translate it before?

Thanks in advance.

**Comment by ulumulu1510** [comment]
There are no language files for Factions or MassiveCore.

To translate everything to spanish you would need to translate both MassiveCore and Factions. There are two ways to do this:
- You can fork them and modify all messages being sent. Please note that not all our messages are simple plain text, but also composed together of different pieces (auto generated names, numbers,  etc) and even so called msons, which allow hover and click effects for messages.
If you get to know our plugins this well to do this, consider the second option.
- Be willing to work with us and supply a PR that creates a language file (in the same way our configuration files work) for both MassiveCore and Factions. This would be easier in the longrun than forking and compilig it yourself and make a lot of people happy.

There are people who have forked it already and translated it, yet I do not remember their names.

Please consider the second option, it would benefit the community greatly and take care of a task we simply don't have time to code ourselfs.

**Comment by ExtremoDuro** [comment]
I agree that creating localization files would be the best in the longrun, but as I am a Java beginner, I don't know which is the best form to do it (Maybe the simplest is Java ResourceBundle?). 

It is not the same to write some crappy code or "fix" for a server than for a public repository where thousands of people download it.

**Comment by ulumulu1510** [comment]
Even as a beginner it should be made clear that a "fix" is not a "solution".

If you are willing to work with us, I would be able to help you with the format and techniques unknown to you. Do not worry about the best way to store such a file (I would take care of that) as our data storage system is easy to use.

**Comment by ExtremoDuro** [comment]
Ok, I will try my best.
